crypto-project-app/src/Item.tsx

```diff
@@ -0,0 +1,10 @@
+const Item =(props)=>(
```

Comment:
Zobacz na moim przykładzie jak zdefiniować prop-sy w react + typeScript i spróbuj to zastosować:

```
import { FC } from "react";

interface CurrencyListItemProps {
  currency: string;
  selectCurrency: (value: string) => void;
}

export const CurrencyListItem: FC<CurrencyListItemProps> = ({
  currency,
  selectCurrency,
}) => {
  return (
    //... kod JSX/TSX/HTML komponentu
  );
};